### ISSUE TITLE
CPU benckmark result doesn't tell if is it good or bad

### ISSUE THREAD

**Issue by AhmedHanafy725** [threefoldtech/zos#2112]
Env: devnet

### Steps

- Get CPU benchmark test result through RMB call `zos.perf.get_all`.

### Result

The result says some numbers but what do these numbers represent? Or what are the good ranges?

```
{
  name: 'CPUBenchmark',
  timestamp: 1700028004,
  result: { multi: 59.723, single: 2.06, threads: 24, workloads: 0 }
}
```

**Comment by muhamadazmy** [comment]
I suggest that a test Task should also have a "description" that explain what do the values mean

**Comment by Omarabdul3ziz** [comment]
basically, the test runs the binary from this [repo](https://github.com/threefoldtech/cpu-benchmark-simple) and as I understand from the code there, it just runs a CRC64 computing process and calculates the timespent. so these numbers represent the time in seconds the less time it takes the better.

verbose logs:
![image](https://github.com/threefoldtech/zos/assets/32527347/0a16b704-bc06-4afc-ac05-4860f2e1bcef)

I did some research for any reference of running `crc64` on a set of cpus but couldn't find any.

**Comment by muhamadazmy** [comment]
We need explanation (docs) on what that score mean. What is a good score and what is a bad score. etc.. create an issue on the cpu-benchmark repo and assign to maxime to give an explanation

**Comment by Omarabdul3ziz** [comment]
check maxime comments on the issue https://github.com/threefoldtech/cpu-benchmark-simple/issues/1#issuecomment-1820051375

**Comment by khaledyoussef24** [comment]
so after checking maxime comment 
the results of the bench cpu performance refer to the efficency of know cpu 
so 
for node 12 on devnet we get

```
   "description": "Measures the performance of the node CPU by reporting the timespent of computing a task in seconds.",
    "name": "cpu-benchmark",
    "result": {
      "multi": 60.999,
      "single": 2.433,
      "threads": 24,
      "workloads": 67
    }, 
```

which is nearly to the same performance as 
2x Intel(R) Xeon(R) CPU L5640 @ 2.27GHz (Q1'2010)

with specs 
```
Single thread score: 2.504
Multi threads score: 72.452 [24 threads]
```

so it's more clear after reading the examples maxime provided 

can this example be added to the docs so it will be more clear for those who run the test ?

**Comment by ramezsaeed** [comment]
Please add the clarification and existing example to the docs, so users can use to compare results.

**Comment by khaledyoussef24** [comment]
solved in another pr #2149
